sys/net/iflib.c

```diff
@@ -6468,7 +6468,11 @@ iflib_msix_init(if_ctx_t ctx)
 	queuemsgs = msgs - admincnt;
 #endif
 #ifdef RSS
-	queues = imin(queuemsgs, rss_getnumbuckets());
+	if (rss_get_enabled() == 0) {
+		queues = queuemsgs;
+	} else {
+		queues = imin(queuemsgs, rss_getnumbuckets());
+	}
```

Comment:
```suggestion
	queues = imin(queuemsgs, rss_get_enabled() == 0 ? queuemsgs : rss_getnumbuckets());
```

a little bit flatter? :)

Reply:
Way better ;)